support-frontend/assets/helpers/abTests/abtest.js

```diff
@@ -137,6 +137,22 @@ function getParticipationsFromUrl(): ?Participations {
   return null;
 }
 
+function getIsRemoteFromAcquisitionData(): boolean {
+  const queryString = getQueryParameter('acquisitionData');
+
+  if (!queryString) {
+    return false;
+  }
+
+  try {
+    const data = JSON.parse(queryString);
+    return data ? data.isRemote === 'true' : false;
```

Comment:
I guess I don't have much context on the other side of this (frontend?), but can we not model it as an A/B test in the acquisition data parameter?

Reply:
@jfsoul yeah, so really this is like a meta-test that wraps a bunch of others. Ideally, we'd store `isRemote` directly in the data lake, but that requires a fair bit of work, so we are using the fact that multiple ab tests are allowed/ab test is a bit like a key-value store in the data lake instead.

An alternative would be to amend the abTest parameter in acquisitionsData to support multiple values, but that's more work and riskier so decided against given this is meant to be temporary.

(But if useful, might be easier to discuss via video!)

Reply:
Ok, that makes sense - I think as a temporary solution this is more acceptable.